<h1>Assignment 1</h1>

<h3>Akash Duttachowdhury</h3>
<h3>21052386</h3>

<b>1. Write a python program to input two numbers and do all basic arithmetic operations on them.</b>

In [1]:
# Akash Duttachowdhury | 21052386
a = int(input("Enter first no.: "))
b = int(input("Enter second no.: "))
print(f"a = {a} & b = {b}")
print(f"a + b = {a+b}")
print(f"a - b = {a-b}")
print(f"a * b = {a*b}")
print(f"a / b = {a/b}")

Enter first no.:  12
Enter second no.:  10


a = 12 & b = 10
a + b = 22
a - b = 2
a * b = 120
a / b = 1.2


<b>2. Write a python program to input two number and swap their values without using any third variable.</b>

In [2]:
# Akash Duttachowdhury | 21052386
a = int(input("Enter a: "))
b = int(input("Enter b: "))
a = a + b
b = a - b
a = a - b
print(f"a = {a}, b = {b}")

Enter a:  2
Enter b:  3


a = 3, b = 2


<b>3. Write a python program to input the temperature in Fahrenheit and change it to Celsius.</b>

In [3]:
# Akash Duttachowdhury | 21052386
f = float(input("Enter temperature in Fahreinheit: "))
c = (f-32)*5/9
print(f"Temperature = {c} C")

Enter temperature in Fahreinheit:  100


Temperature = 37.77777777777778 C


<b>4. Write a python program to input the basic salary of a person and compute its TA (20% of
basic), DA (120% of basic), HRA (30% of basic), Gross (basic + ta + da + hra).</b>

In [4]:
# Akash Duttachowdhury | 21052386
basic = int(input("Enter the basic salary: "))
print(f"TA: Rs. {0.2*basic}")
print(f"DA: Rs. {1.2*basic}")
print(f"HRA: Rs. {0.3*basic}")
print(f"Gross: Rs. {2.7*basic}")

Enter the basic salary:  10000


TA: Rs. 2000.0
DA: Rs. 12000.0
HRA: Rs. 3000.0
Gross: Rs. 27000.0


<b>5. Write a python program to swap three variables.

In [5]:
# Akash Duttachowdhury | 21052386
print("Enter three no.s")
a = int(input())
b = int(input())
c = int(input())
temp = a
a = b
b = c
c = temp
print(f"After swapping\na = {a}, b = {b}, c = {c}")

Enter three no.s


 3
 4
 5


After swapping
a = 4, b = 5, c = 3


<b>6. Write a python program to evaluate the expression
4x^4 + 3y^3 -9z + 6

In [6]:
# Akash Duttachowdhury | 21052386
x = int(input("Enter x: "))
y = int(input("Enter y: "))
z = int(input("Enter z: "))
print(f"Evaluated expression = {4*x**4 + 3*y**3 - 9*z + 6}")

Enter x:  8
Enter y:  4
Enter z:  2


Evaluated expression = 16564


<b>7. Write a python program to take a input in uppercase and change it to lower case

In [7]:
# Akash Duttachowdhury | 21052386
l = str(input("Enter a string: "))
print(l.lower())

Enter a string:  UPPER CASE STRING


upper case string


<b>8. Write a python program to input the radius of a circle and print its area and perimeter.

In [8]:
# Akash Duttachowdhury | 21052386
from math import pi
r = float(input("Enter radius of circle: "))
print(f"Area of circle is {pi*r**2}")

Enter radius of circle:  18


Area of circle is 1017.8760197630929


<b>9. Write a python program to input marks in 5 subjects of a student and print its average mark.

In [9]:
# Akash Duttachowdhury | 21052386
print("Enter the marks for 5 students")
m1 = int(input())
m2 = int(input())
m3 = int(input())
m4 = int(input())
m5 = int(input())
print(f"Avg marks = {(m1+m2+m3+m4+m5)/5}")

Enter the marks for 5 students


 88
 99
 78
 89
 97


Avg marks = 90.2


<b>10. Write a python program to input a number and print its square, cube and fourth power.

In [10]:
# Akash Duttachowdhury | 21052386
num = int(input("Enter a number: "))
print(f"Square = {num**2}\nCube = {num**3}\nFourth = {num**4}")

Enter a number:  3


Square = 9
Cube = 27
Fourth = 81


<b>11. Write a python program to input a the sides of a triangle and print its area.

In [11]:
# Akash Duttachowdhury | 21052386
import math as m
print("Enter the sides of a triangle")
a = int(input("a = "))
b = int(input("b = "))
c = int(input("c = "))
s = (a+b+c)/2
print(f"Area of triangle = {m.sqrt(s*(s-a)*(s-b)*(s-c))}")

Enter the sides of a triangle


a =  3
b =  4
c =  5


Area of triangle = 6.0


<b>12. Write a python program to compute SI and CI.

In [12]:
# Akash Duttachowdhury | 21052386
p = float(input("Enter principal: "))
r = float(input("Enter rate(%): "))
t = float(input("Enter time(in years): "))
n = int(input("Enter no. of times interest is compounded per year: "))
print(f"Simple Interest = Rs. {p*r*t}")
print(f"Compound Interest = Rs. {p*((1+r/n)**(n*t))}")

Enter principal:  1000
Enter rate(%):  3
Enter time(in years):  12
Enter no. of times interest is compounded per year:  1


Simple Interest = Rs. 36000.0
Compound Interest = Rs. 16777216000.0


<b>13. Ask the user to enter a number x. Use the sep optional argument to print out x, 2x, 3x, 4x,
and 5x, each separated by three dashes, like below.
Enter a number: 7
7---14---21---28---35

In [13]:
# Akash Duttachowdhury | 21052386
n = int(input("Enter number: "))
print(f"{n}", f"{2*n}", f"{3*n}", f"{4*n}", f"{5*n}", sep='---')

Enter number:  7


7---14---21---28---35
